WotLK/Ahnkahet/Trash.lua

```diff
@@ -0,0 +1,79 @@
+--------------------------------------------------------------------------------
+-- Module Declaration
+--
+
+local mod, CL = BigWigs:NewBoss("Ahn'kahet Trash", 522)
+if not mod then return end
+mod.displayName = CL.trash
+mod:RegisterEnableMob(
+	30278, -- Ahn'kahar Spell Flinger
+	30285, -- Eye of Taldaram
+	30319 -- Twilight Darkcaster
+)
+
+--------------------------------------------------------------------------------
+-- Localization
+--
+
+local L = mod:GetLocale()
+if L then
+	L.spellflinger = "Ahn'kahar Spell Flinger"
+	L.eye = "Eye of Taldaram"
+	L.darkcaster = "Twilight Darkcaster"
+end
+
+--------------------------------------------------------------------------------
+-- Initialization
+--
+
+function mod:GetOptions()
+	return {
+		--[[ Ahn'kahar Spell Flinger ]]--
+		59102, -- Shadow Blast
+
+		--[[ Eye of Taldaram ]]--
+		56728, -- Eyes in the Dark
```

Comment:
"Eyes in the Dark" isn't very descriptive, it's basically as 10s debuff that can interrupt you randomly,